metrics/core/src/test/java/org/hyperledger/besu/metrics/opentelemetry/OpenTelemetryMetricsSystemTest.java

```diff
@@ -175,12 +172,9 @@ public void shouldCreateObservationFromGauge() {
     final ObservableMetricsSystem localMetricSystem =
         MetricsSystemFactory.create(metricsConfiguration);
     localMetricSystem.createGauge(RPC, "myValue", "Help", () -> 7d);
-    List<MetricData.ValueAtPercentile> values = new ArrayList<>();
-    values.add(MetricData.ValueAtPercentile.create(0, 7d));
-    values.add(MetricData.ValueAtPercentile.create(100, 7d));
 
     assertThat(localMetricSystem.streamObservations())
-        .containsExactlyInAnyOrder(new Observation(RPC, "myValue", values, emptyList()));
+        .containsExactlyInAnyOrder(new Observation(RPC, "myValue", 7d, emptyList()));
```

Comment:
style nit:  `7.0` would be less likely to be confused with hex encoded 125.

Reply:
`d` stands for double - I'll edit to make sure it's all .0 instead, it's indeed surprising.

Reply:
Done

Reply:
Some style guides I've seen insist on one case for `0xhex` encoded letters and another case for type designators.  Typically capitals for the types and lower case for actual digits.